couchbase-sdk-cxx-black-duck-manifest.yaml

```diff
@@ -2,13 +2,13 @@
 components:
   boringssl:
     bd-id: 1e303b15-b82b-4a4d-8cd6-a2048fb2c73d
-    versions: [ 20200921 ]
+    versions: [ 20240201 ]
```

Comment:
I don't think we want to change this value.  See CBD-5677 for details outlining why we chose the previous value (20200921).  It has to do w/ what is in the blackduck knowledge base.  I think if we want to try and change we need to make sure the BD scans are okay.

Reply:
I have updated ticket to point to https://github.com/couchbaselabs/couchbase-cxx-client/pull/544